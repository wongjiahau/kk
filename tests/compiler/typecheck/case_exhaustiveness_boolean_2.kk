let and = 
  \(true, true) => true
  \(false, true) => false
