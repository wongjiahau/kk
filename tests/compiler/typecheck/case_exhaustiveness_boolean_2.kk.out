1
========

========
error:
  ┌─ tests/compiler/typecheck/case_exhaustiveness_boolean_2.kk:2:3
  │
2 │ ╭   \(true, true) => true
3 │ │   \(false, true) => false
  │ ╰─────────────────────────^ Non-exhaustive cases
  │
  = Missing case(s):

      \(true, false) => ...
      \(false, false) => ...
